api/handlers.go

```diff
@@ -18,6 +18,7 @@ import (
 	"github.com/docker/docker/pkg/parsers/kernel"
 	versionpkg "github.com/docker/docker/pkg/version"
 	apitypes "github.com/docker/engine-api/types"
+	"github.com/docker/engine-api/types/container"
```

Comment:
I call this `containertypes` because `container` is an overloaded word in this package :)